Extensions/blacklist.js

```diff
@@ -176,6 +176,16 @@ XKit.extensions.blacklist = new Object({
 
 		}
 
+		if (XKit.page.react) {
+			const react_css =
+				'.xblacklist_excuse { ' +
+				'padding: 15px 0 23px 0;' +
+				'height: 50px;' +
+				'background-color: white;' +
```

Comment:
indentation pls
(this would probably be nicer in a template string)

Reply:
I thought there was something cool like this, thanks for pointing it out!